packages/@react-spectrum/s2/src/AvatarGroup.tsx

```diff
@@ -114,32 +116,28 @@ const AvatarGroupContext = createContext<AvatarGroupContextProps>({});
 const container = style({
   display: 'flex',
   alignItems: 'center',
-  listStyleType: 'none',
-  paddingStart: 0,
   margin: 0
 }, getAllowedOverrides());
 
 function AvatarGroup(props: AvatarGroupProps) {
   let {children, label, size = 'M', strokeColor = 'base', styles, UNSAFE_style, UNSAFE_className, ...otherProps} = props;
+  let groupId = useId();
   let labelId = useId();
 
   return (
-    <ul
-      aria-labelledby={labelId}
-      {...filterDOMProps(otherProps)}
-      tabIndex={0}
-      role="group"
-      className={(UNSAFE_className ?? '') + container(null, styles)}
-      style={UNSAFE_style}>
-      {
-        Children.map(children, (child, index) => (
-          <AvatarGroupContext.Provider value={{styles: avatar({size, isFirst: index === 0}), strokeColor}}>
-            {child}
-          </AvatarGroupContext.Provider>
-        ))
-      }
-      {label && <span id={labelId} className={text({size})}>{label}</span>}
-    </ul>
+    <AvatarGroupContext.Provider value={{styles: avatar({size}), strokeColor}}>
+      <div
+        id={groupId}
+        aria-labelledby={`${groupId} ${labelId}`}
+        aria-label="online members"
```

Comment:
this should be a prop, we don't know if it'll always be online members

Reply:
ah whoops, used this to test and accidentally committed it. will remove

Reply:
If the containing group with `id={groupID}` does not have an `aria-label`, or some default label, like `Avatar Group`, we should not include `groupId` in the `aria-labelledby`, otherwise the group will include the accessibility name for all the descendants, including the label in the accessibility name, which may lead to redundancy and verbosity.